modules/node_modules/@ncigdc/dux/cart.js

```diff
@@ -92,6 +92,29 @@ function toggleFilesInCart(incomingFile) {
   };
 }
 
+function removeFilesFromCart(files) {
+  return (dispatch, getState) => {
+    const filesToRemove = _.isArray(files) ? files : [files];
+    const existingFiles = getState().cart.files;
+    const nextFiles = _.differenceBy(existingFiles, filesToRemove, 'file_id');
+    const filesRemoved = existingFiles.length - nextFiles.length;
+    dispatch(notify(getNotificationComponent('remove', `remove/${filesToRemove.file_name}`, {
```

Comment:
For the unique ids, I think we could generate that in the `getNotificationComponent`. I'll check with the others to see if that'd ok